Engines/Wine/Verbs/xact/script.js

```diff
@@ -0,0 +1,173 @@
+include(["Engines", "Wine", "Engine", "Object"]);
+include(["Utils", "Functions", "Net", "Resource"]);
+include(["Utils", "Functions", "Filesystem", "Files"]);
+
+/**
+* Inspired from d3dx9 verb, winetricks xact
+* -> https://github.com/Winetricks/winetricks/blob/63bc6dbe612d017a0cb6bf6e4cde265162d75bca/src/winetricks#L9643
+* @returns {Wine}
+*/
+Wine.prototype.xact = function () {
+    var that = this;
+
+    var extractXacttoSystemDirectory = function(progressBar, filesToExtract, destination, pattern, directory) {
```

Comment:
The `filesToExtract` and `destination` are given as parameters. Therefore I think the function name is more complex than it has to be. I would prefer a more simple name like `extractFiles`. Also, please add some documentation for the function.

Reply:
It's basically "extractDirectXtoSystemDirectory" from the directx9 verb